mypy.ini

```diff
@@ -20,6 +20,18 @@ warn_return_any          = True
 warn_unreachable         = True
 warn_unused_ignores      = True
 
+# These are too strict for us at the moment
```

Comment:
"... but we should gradually enable them as time allows" :)

Reply:
"at the moment" :-)